src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -208,6 +209,18 @@ public void editPbaScenarios() {
     public void deleteOrganisationScenarios() {
         deletePendingOrganisationShouldReturnSuccess();
         deleteActiveOrganisationShouldReturnSuccess();
+        deleteDxAddressShouldReturnSuccess();
+    }
+
```

Comment:
add negative tests to cover exceptions from eg empty contact lists empty dx addresses etc